packages/expo-cli/src/commands/eas-build/build/action.ts

```diff
@@ -205,6 +211,11 @@ async function startBuildAsync<T extends Platform>(
     const job = await builder.prepareJobAsync(archiveUrl);
     log(`Starting ${platformDisplayNames[job.platform]} build`);
 
+    if (process.env.EAS_OUTPUT_JOB_JSON === '1') {
+      process.stdout.write(`>>> EAS_JOB_START\n${JSON.stringify(job)}\n>>> EAS_JOB_END`);
```

Comment:
Why do you need the delimiters (EAS_JOB_START and EAS_JOB_END)?

Reply:
Why not `log`?

Reply:
@dsokal Just want to be explicit that the output is written to stdout so we can easily pipe it for testing.